### PR TITLE
chore: Add editorconfig and prettier

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

**Comment by oed** [comment]
I see that we have the story here: https://github.com/ceramicnetwork/js-ceramic/issues/879
When you open PRs agains stories please connect it to the story using the zenhub plugin:
image

File: package.json [hunk 1]
```diff
@@ -25,11 +25,17 @@
...
     "ts-node": "^9.0.0",
     "typedoc": "^0.19.2",
     "typedoc-plugin-lerna-packages": "^0.3.1",
     "typedoc-plugin-pages": "^1.0.1",
     "typescript": "^4.0.5"
+  },
+  "prettier": {
+    "printWidth": 120,
+    "trailingComma": "all",
+    "singleQuote": true
```
Comment (oed) [inline package.json:39]: Would be nice if we could start using the https://github.com/3box/eslint-config-3box package for this entire repo. But that is likely a much larger change.

**Review by oed — APPROVED** [review]

**Comment by ukstv** [comment]
Okay.

ZenHub lacks Safari plugin. So sad.

**Comment by oed** [comment]
That's unfortunate. It works for Firefox and chromium. Does safari actually have a plugin system?

**Comment by oed** [comment]
You should be able to connect it within the zenhub app as well fyi :)

File: package.json (hunk 1 above)
Comment (ukstv) [reply package.json:39]: It definitely is bigger. 3box eslint rules are too strict for this code 😉

**Pull request merged**